amida-tech/orange-api
# Task/auto incrementing

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do?
Adds docs pertaining to auto-incrementing ids, updates load-testing script to test speed of inserts into patients table (which requires using the auto-incrementing id functionality). Most importantly provides a fleshed out seed script type file that generates two users on the orange api and creates a conversation between them to be visualized in the Messages Screen of the Orange Client. 

#### Related JIRA tickets:
https://jira.amida-tech.com/browse/ORANGE-217
https://jira.amida-tech.com/browse/ORANGE-223
https://jira.amida-tech.com/browse/ORANGE-181

#### How should this be manually tested?
To test the mockScript.js `cd` into your `orange-api` directory and run `node mockScript.js` from the command line. You should then see printed out two different logins for the app ( the passwords are both `testtest`) and times at which the messages b/w them where created. Go into the orange client and check the messages view. 

#### Background/Context

#### Screenshots (if appropriate):
